app/views/cannot-apply-exception.njk

```diff
@@ -1,111 +1,130 @@
-        <p class="govuk-body">You can <a href='{{ backLink }}'>apply for another business</a> that you have permission to act for.</p>
+        {% if hasMultipleBusinesses %}
+          <h2 class="govuk-heading-m">Apply for another business</h2>
+          <p class="govuk-body">You can <a href='{{ backLink }}'>apply for another business</a> that has not already applied.</p>
+        {% endif %}
       {% endif %}
 
-      <h2 class="govuk-heading-m">Get help with your application</h2>
-      <p class="govuk-body">If you need help with your application, contact the Rural Payments Agency by email or phone.</p>
-      {{ rpaContactDetails({email: ruralPaymentsAgency.email, callChargesUri: ruralPaymentsAgency.callChargesUri, telephone: ruralPaymentsAgency.telephone}) }}
-    {% endif %}
-
-    {% if cphError %}
-      <h1 class="govuk-heading-l">You cannot apply for reviews or follow-ups for this business</h1>
-      <p class="govuk-body">{{ organisationName }}{{ sbiText }} has no eligible county parish holding (CPH) number registered to it.</p>
-    
-      <p class="govuk-body">To be eligible a CPH must be registered in England.</p>
-      <p class="govuk-body">The CPH number cannot be:</p>
-      <ul class="govuk-list govuk-list--bullet">
-        <li>an 8000 number used for abattoirs, pet food manufacturers, racing pigeon establishments and showgrounds</li>
-        <li>a 9000 number used for poultry keepers</li>
-      </ul>
-
-      {% if hasMultipleBusinesses %}
-        <h2 class="govuk-heading-m">Apply for another business</h2>
-        <p class="govuk-body">You can <a href='{{ backLink }}'>apply for another business</a> that does have an eligible CPH.</p>
+      {% if noEndemicsAgreementError %}
+        <a class="govuk-back-link" href='{{backLink}}'>Back</a>
+        <h1 class="govuk-heading-l">You do not have an agreement for this business</h1>
+        <p class="govuk-body">The {{ organisationName }} – {{ sbiText }} does not have an agreement for this service.</p>
+        <h2 class="govuk-heading-m">What to do next</h2>
+        <p class="govuk-body">Before you can claim you need to:</p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>
+            <a href="#">apply for funding to improve livestock health and welfare</a>
+          </li>
+          <li>read the guidance on <a href="#">how to claim funding</a>
+          </li>
+        </ul>
+        <h2 class="govuk-heading-m">Get help with your application</h2>
+        <p class="govuk-body">If you need help with your application, contact the Rural Payments Agency by email or phone.</p>
+        {{ rpaContactDetails({email: ruralPaymentsAgency.email, callChargesUri: ruralPaymentsAgency.callChargesUri, telephone: ruralPaymentsAgency.telephone}) }}
+        <h2 class="govuk-heading-m">Claim for another business</h2>
+        <p class="govuk-body">You can claim <a href='{{ backLink }}'>for another business</a> that has an agreement.</p>
```

Comment:
I believe this bit should be for multiple business only: {% if hasMultipleBusinesses %}